pkg/oauth/apis/oauth/types.go

```diff
@@ -4,6 +4,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
@deads2k does OAuthAccessToken implement full CRUD? I see this in legacy:

```
  Create(token *oauthapi.OAuthAccessToken) (*oauthapi.OAuthAccessToken, error)
  Get(name string, options metav1.GetOptions) (*oauthapi.OAuthAccessToken, error)
  List(opts metav1.ListOptions) (*oauthapi.OAuthAccessTokenList, error)
  Delete(name string) error
```

No watch, no patch, no update :-) (genclient:onlyVerbs=create,get,list,delete) ?